README.md

```diff
@@ -1,73 +1,70 @@
 # C4C Internal Backend
 
-Code 4 Community internal website backend to manage logins, meeting attendance, and requests from other organizations. Created using Vert.x in Java.
+Code 4 Community internal website backend to manage logins, event attendance, and requests from other organizations. Created using Vert.x in Java.
 
 ## API Endpoints
 
 ### Authorization
-For any request to protected resources you should pass the JWT in the format of this string in the HTTP header Authorization as: 
-	
-	Authorization: Bearer <token>
 
+For any request to protected resources you should pass the JWT in the format of this string in the HTTP header Authorization as:
+Authorization: Bearer <token>
 
 ### Endpoints
+
 Note that routes that should modify data ought to be POST requests, this is being changed soon.
 
 ```sh
 GET https://localhost:8443/signup?username=<username>&password=<password>
 ```
-- Creates a member in the database the given username and password
 
+- Creates a user in the database the given username and password
 
 ```sh
 GET https://localhost:8443/login?username=<username>&password=<password>
 ```
-- Returns JWT in response header under "Authorization". 
 
+- Returns JWT in response header under "Authorization".
 
 ```sh
 GET https://localhost:8443/logout
 ```
-- Given a valid JWT through the request header, will invalidate the JWT on the server-side
 
+- Given a valid JWT through the request header, will invalidate the JWT on the server-side
 
 ```sh
-GET https://localhost:8443/api/v1/members/
+GET https://localhost:8443/api/v1/users/
 ```
-- Returns a list of all of our members.
 
+- Returns a list of all of our users.
 
 ```sh
-GET https://localhost:8443/protected/createmeeting?id=<id>&name=<name>&date=<yyyy-MM-dd HH-mm>&open=<open>
+GET https://localhost:8443/protected/createevent?id=<id>&name=<name>&date=<yyyy-MM-dd HH-mm>&open=<open>
```

Comment:
Try to rewrite this section to make it very clear what data type each of those query params are (is id an integer or a string?).

For example you can try specifying their types in the query params as `?id=UUID&name=STRING&...` and then give their interpretation below